### ISSUE TITLE
Error installing on Debian

### ISSUE THREAD

**Issue by ychae** [Sage-Bionetworks/synapser#207]
## Operating system
Distributor ID: Debian
Description:    Debian GNU/Linux 9.6 (stretch)
Release:        9.6
Codename:       stretch

## Description of the problem
When I run 
```
install.packages("synapser", repos=c("https://sage-bionetworks.github.io/ran", "http://cran.fhcrc.org"))
```
in RStudio, synapser fails to install. 

### Actual behavior

Error message:
```
error: The 'keyrings.alt' distribution was not found and is required by synapseclient
Warning in install.packages :
  installation of package ‘synapser’ had non-zero exit status
```

## Output of `sessionInfo()`
```
> sessionInfo()
R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Debian GNU/Linux 9 (stretch)

Matrix products: default
BLAS: /usr/lib/openblas-base/libblas.so.3
LAPACK: /usr/lib/libopenblasp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8       
 [4] LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C              
[10] LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_3.5.1 tools_3.5.1    yaml_2.2.0   
```

**Comment by ychae** [comment]
@kimyen Thanks for the update! I tried @kdaily's workaround and installed `keyrings.alt` as root but still wasn't able to get `synapser` to install.

**Comment by kimyen** [comment]
Documenting my discussion with @kdaily. Everyone who is using Linux probably is running into this problem: https://sagebionetworks.jira.com/browse/SYNR-1375
The fix is scheduled to be released later this month. @ychae , if you need a workaround asap, we can setup a virtual meet and I can walk you through instructions on getting a working installation. Though if there are more people need to install synapser on a Linux environment, I would recommend waiting for the fix.

**Comment by ychae** [comment]
Thanks for the update @kimyen! I might need the workaround ASAP. I’ll contact you offline for more details. Thanks again! :)

**Comment by kimyen** [comment]
This issue has been fixed in synapser 0.5
